CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 Running changelog of releases since `1.0.0-beta01`
 
+## v3.0.1
```

Comment:
Shouldn't we also update the project version?

Reply:
Good point, thanks for reminding me.